projects/netgrif-components-core/src/lib/data-fields/file-field/file-default-field/abstract-file-default-field.component.ts

```diff
@@ -220,6 +220,12 @@ export abstract class AbstractFileDefaultFieldComponent extends AbstractBaseData
             this.fileUploadEl.nativeElement.value = '';
             return;
         }
+        if (this.dataField.allowTypes && !this.checkTypes(this.fileUploadEl.nativeElement.files.item(0).type)) {
```

Comment:
An undefined check or null check would be reuquired, please consider to use ``?.`` operator.

Reply:
done